### DISCUSSION TITLE
开放题评分后如何通知到用户

### DISCUSSION THREAD

**Discussion by deanplus** [decert-me/website#289] [General]
目前，只有页面上有通知。要不要加上其他通知方式，比如邮箱、Discord？

**Comment by xilibi2003** [comment]
各有利弊：
邮箱触达率应该好一些。
Discord 可以增加互动氛围。

**Reply by deanplus** [reply]
邮箱触达率是更好，但需要用户绑定邮箱；Discord 估计很多人看不到或不看
